src/rpc/common/AnyHandler.h

```diff
@@ -80,13 +80,30 @@ class AnyHandler final
         return pimpl_->process(value);
     }
 
+    /**
+     * @brief Process incoming JSON by the stored handler in a provided
+     * coroutine
+     *
+     * @param value The JSON to process
+     * @return JSON result or @ref RPC::Status on error
+     */
+    [[nodiscard]] ReturnType
+    process(
+        boost::json::value const& value,
+        boost::asio::yield_context& ptrYield) const
+    {
+        return pimpl_->process(value, &ptrYield);
```

Comment:
I'm ok with this because the intention is to remove `yield` later anyways but:
I would add overloads everywhere down to the concept check instead of passing a pointer down the line.
See below comment for why that is.

Reply:
Discussed offline. We decide not to add the override all  the way to concept for now.